server/routes/v1/posts.js

```diff
@@ -17,4 +17,12 @@ router.patch("/posts", middleware.apiAuth, exists, post.updatePost);
 
 router.delete("/posts", middleware.apiAuth, exists, post.deleteById);
 
+// post activity
+router.get("/posts/:post_id/activity", post.activity.get)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/logchimp/logchimp/pullRequest?prid=7771592)
```suggestion
router.get("/posts/:post_id/activity", post.activity.get);
```